src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -317,22 +317,19 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
   };
 
   setCustomClasses = () => {
+    const classesAndConditions = [
+      ...(this.props.metadataClassesConditions || []),
+    ];
     if (this.props.isCurrentAsset) {
-      this.addCssClassesToMetadataContainer({
+      classesAndConditions.push({
         condition: this.props.isCurrentAsset,
         className:
           currentAssetClassName ||
           (this.props.customClassNames || {}).currentAsset,
```

Comment:
Could this code be improved?  It looks like `className` could be assigned `undefined` in case `this.props.customClassNames` is falsy

Reply:
haha, good cath, this code is actually never even executed, because currentAssetClassName is const that is always defined. Will fix

Reply:
Fixed and added tests